src/coreclr/pal/src/configure.cmake

```diff
@@ -22,6 +22,9 @@ endif()
 
 if(CLR_CMAKE_TARGET_LINUX AND NOT CLR_CMAKE_TARGET_ANDROID)
   set(CMAKE_RT_LIBS rt)
+  if(CLR_CMAKE_HOST_ARCH_LOONGARCH64)
```

Comment:
Unless this results in build breakage, I would move changes from this file to https://github.com/dotnet/runtime/pull/62887 that way each PR would be self-contained.

Reply:
Thanks.
Had moved.

Reply:
Thanks.
Had moved.

Reply:
Thanks.
Had moved.